docs/advanced_topics/customisation/custom_page_listings.md

```diff
@@ -0,0 +1,64 @@
+# Custom page listings
```

Comment:
Could we have a corresponding entry for `PageListingViewSet` in https://github.com/wagtail/wagtail/blob/main/docs/reference/viewsets.md?plain=1, and reword one of the sentences here to explicitly mention `PageListingViewSet` with a link to the section in the API reference?

With enough docstrings in the source code, this should be just autodoc directives for the most part (if not all).

Then, add an explicit target so we can link to this from the reference docs.

```suggestion
(custom_page_listings)=
# Custom page listings
```

Reply:
Thanks @laymonage! Have added the reference docs in 2de9bd66cb.